packages/xo-web/src/xo-app/xoa/support/index.js

```diff
@@ -0,0 +1,66 @@
+import _ from 'intl'
+import ActionButton from 'action-button'
+import decorate from 'apply-decorators'
+import React from 'react'
+import { adminOnly } from 'utils'
+import { Card, CardBlock, CardHeader } from 'card'
+import { Container, Row, Col } from 'grid'
+import { injectState, provideState } from 'reaclette'
+import { getXoaCheck } from 'xo'
+
+const Support = decorate([
+  adminOnly,
+  provideState({
+    initialState: () => ({ stdoutXoaCheck: [], stderrXoaCheck: [] }),
+    effects: {
+      async getXoaCheck() {
+        const result = await getXoaCheck()
+        const { stdout, stderr } = result
+        return {
+          stdoutXoaCheck:
+            stdout !== undefined ? stdout.split('\n') : result.split('\n'),
+          stderrXoaCheck: stderr !== undefined ? stderr.split('\n') : [],
+        }
+      },
+    },
+  }),
+  injectState,
+  ({ effects, state: { stderrXoaCheck, stdoutXoaCheck } }) => (
+    <Container>
+      <Row>
+        <Col mediumSize={6}>
+          <Card>
+            <CardHeader>{_('checkXoa')}</CardHeader>
+            <CardBlock>
+              <ActionButton
+                btnStyle='success'
+                handler={effects.getXoaCheck}
+                icon=''
+              >
+                {' '}
+                {_('checkXoa')}
+              </ActionButton>
+              <hr />
+              <span className='text-danger'>
+                {stderrXoaCheck.map((err, key) => (
+                  <div key={key}>
+                    <span>{err}</span>
+                  </div>
```

Comment:
Given that erros always objects, it's better to display the the original format using `<pre>` HTML element
```js
 <pre key={key}>{err}</pre>
```

Reply:
and for homogeneity with what we did in the updater page :+1: